Dialog/CreditEditor_TableView.h

```diff
@@ -0,0 +1,32 @@
+﻿#ifndef CUSTOMQTABLEVIEW_H
```

Comment:
Let's name this preprocessor directive consistently with the file name (see other headers for example)